### PR TITLE
Add step by step guide for local development with tilt

### PR DESCRIPTION
**Pull request by faizanahmad055**
Signed-off-by: faizanahmad055 <faizan.ahmad55@outlook.com>

### CODE REVIEW

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/161628fd9d08956e67dec2951b750e57e5d7c315/checks) has Failed!

File: content/sre/local-development/tilt/step-by-step-guide.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+```bash
+helm dependency update
+
+cd ..
+```
+
+7) Add Tiltfile to your application 
+
+[Here](Tiltfile) is a sample Tiltfile for a Java based project with a specific Dockerfile.
```
Comment (rasheedamir) [inline content/sre/local-development/tilt/step-by-step-guide.md]: @faizanahmad055 link dockerfile here

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/537a26a646e46ebf69ac636fa66c39180fb6fe37/checks) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-233-48220598`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.170`

**Pull request merged**